tests/30rooms/60anonymousaccess.pl

```diff
@@ -756,3 +754,28 @@ sub ignore_presence_for
       )
    } @events ];
 }
+
+sub expect_4xx_or_empty_chunk
+{
+   my ( $f ) = @_;
+   $f->then( sub {
+      my ( $body ) = @_;
+
+      log_if_fail "Body", $body;
+
+      assert_json_keys( $body, qw( chunk ) );
+      assert_json_list( $body->{chunk} );
+      die "Want list to be empty" if @{ $body->{chunk} };
+
+      Future->done( 1 );
+   },
+   http => sub {
+      my ( undef, undef, $response ) = @_;
+
+      log_if_fail "Response", $response;
```

Comment:
Put "HTTP response" in the message, as usually we call "Response" the decoded JSON body of a 200 OK message.